src/pages/VsCodeExtension.vue

```diff
@@ -194,6 +195,18 @@ export default {
         );
       },
     },
+    '$store.getters.focusedEvent': {
+      handler(event) {
+        if (event) {
+          this.setView(VIEW_FLOW);
+          window.requestAnimationFrame(() => {
```

Comment:
I'm not sure if this is reliable. If you're waiting for the DOM to update, consider `this.$nextTick(() => { ... })`?
https://vuejs.org/v2/api/#Vue-nextTick

Reply:
Yes, inside Vue component it's better to use Vue methods.
Done: https://github.com/applandinc/appmap-js/pull/168/commits/41108ac4f7366b2505d5d038559cd59f32b6a2ce